nova/api/openstack/placement/objects/resource_provider.py

```diff
@@ -2749,16 +2752,19 @@ def _get_usages_by_provider_tree(ctx, root_ids):
                        _RP_TBL.c.id.in_(root_ids))
                 )
     )
-    usage = sa.alias(
-        sa.select([
-            _ALLOC_TBL.c.resource_provider_id,
-            _ALLOC_TBL.c.resource_class_id,
-            sql.func.sum(_ALLOC_TBL.c.used).label('used'),
-        ]).select_from(derived_alloc_to_rp).group_by(
-            _ALLOC_TBL.c.resource_provider_id,
-            _ALLOC_TBL.c.resource_class_id
-        ),
-        name='usage')
+    usage_sel = sa.select([
```

Comment:
I'd name this variable `usage_select`, I don't see a reason to shorten it.

Reply:
I've seen the same `_sel` shortening across this file so I think that's the reason of shortening it here as well.

Reply:
Ah, okay. If it's consistent with surrounding code, then that's cool.